routing/turns_generator.cpp

```diff
@@ -668,6 +668,8 @@ size_t CheckUTurnOnRoute(TUnpackedPathSegments const & segments,
   auto const & masterSegment = segments[currentSegment - 1];
   if (masterSegment.m_path.size() < 2)
     return 0;
+  if (segments[currentSegment].m_path.empty())
```

Comment:
There's an issue in our system. There are road ids in osrm which don't have appropriate ids in mwm. When the route goes through such osrm id we get     
```
LOG(LERROR, ("Can't unpack geometry for map:", mapping.GetCountryName(), " node: ",
                 osrmPathSegment.node)); 
```
on route reconstruction stage. As a result some item of |segments| vector could have an empty |m_path|. It leads to crashes on release version.

Probably the best solution is to have for every osrm id an appropriate road id in mwm. As far as I remember Lev has investigated the problem and told me that there're some cases when osrm edge cannot have a corresponding road id in mwm. Anyway it's a topic for further investigation.

For the time being our code should rely that some orsm edges doesn't have a corresponding features. (At least in release version.)